tools/clang-tidy-plugin/test/lit.cfg

```diff
@@ -25,7 +25,7 @@ cata_include = os.path.join( config.cata_source_dir, "./src" )
 cata_third_party_include = os.path.join( config.cata_source_dir, "./src/third-party" )
 test_include = os.path.join( config.cata_source_dir, "./tools/clang-tidy-plugin/test-include" )
 
-if config.cata_clang_tidy_executable == "ON":
+if config.cata_clang_tidy_executable in ["TRUE", "ON"]:
```

Comment:
apparently `-DCATA_CLANG_TIDY_EXECUTABLE=ON` results in a different string from setting `"CATA_CLANG_TIDY_EXECUTABLE": true` in the cmake preset. Cmake things.